poleno/mail/tests/test_smtp.py

```diff
@@ -160,6 +160,7 @@ def test_message_with_both_text_and_html_body(self):
 
                 <p>HTML content</p>
                 --===============.*==--
+
```

Comment:
@martinmacko47 Nastala mala zmena iba vo whitespace. Teoreticky aj za poslednou castou patternu `--===============.*==--` by mal nasledovat prazdny riadok, ale `self.assertRegexpMatches` vyuziva `re.search` takze nastane match.

Reply:
@viliambalaz Hej. Mozu byt drobne rozdiely ako to rozne verzie SMTP backendov zaencoduju. Pri upgradoch verzii asi budeme musiet opat obcas nieco opravit.